src/ZODB/Connection.py

```diff
@@ -951,7 +945,7 @@ def _release_resources(self):
                 c._storage.release()
             c._storage = c._normal_storage = None
             c._cache = PickleCache(self, 0, 0)
-            c.transaction_manager = None
+            c.close(False)
```

Comment:
This is 1 of the 3 main changes. I didn't plan to change this at the beginning but while running tests, I discovered that closing a DB left Connections registered to the global transaction manager. Before, these Connections were ignored by patching their afterCompletion/newTransaction.